adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -277,17 +306,32 @@ func CreateRoutesWithClusters(mgwSwagger model.MgwSwagger, upstreamCerts map[str
 				apiTitle, apiVersion, resourcePath)
 		}
 
+		// In case resource level production endpoints exist and no resource level sandbox
+		// use API level sandbox for sandbox route creation
+		if resourceBasePath != "" && resourceBasePathSand == "" {
+			apiLevelbasePathSand = apiLevelbasePath
```

Comment:
Why are we updating `apiLevelbasePathSand` here? or should line 318 change?

Reply:
@suksw In case resource level production endpoints exist and no resource level sandbox endpoint exist for that resource, we fallback to api level sandbox endpoint. For example consider the following example.

```yaml
# API level end points
x-wso2-production-endpoints:
  urls:
    - http://mockBackend:2383/v2
x-wso2-sandbox-endpoints:
  urls:
    - http://mockBackend:2390/v2
     
paths:
  /v2/pets/findByTags:
    # resource level production endpoint
    x-wso2-production-endpoints:
      urls:
        - http://mockBackend:2391/v2prod
```
We consider the resource level production endpoint (http://mockBackend:2391/v2prod) when creating the route for production. For sandbox route we consider API level sandbox endpoint (http://mockBackend:2390/v2) because there's no resource level sandbox endpoint.